docs/develop-resources/guides/4-add-controller.md

```diff
@@ -41,11 +41,16 @@ hack/compare-mock fixtures/<your_resource_test>
 
 ### Existing DCL/TF based resource
 
-```
-KCC_USE_DIRECT_RECONCILERS=<YOUR KIND> hack/compare-mock fixtures/<your_resource_test>
-```
+Copy all existing test cases and add `-direct` suffix to the test names.
+
+Enable direct controller for the the new test cases by setting the annotation `alpha.cnrm.cloud.google.com/reconciler: direct` in `create.yaml` and `update.yaml`. [example](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/c0a77915723788e5068a819c93f22c4661e47b6b/pkg/test/resourcefixture/testdata/basic/dataflow/v1beta1/dataflowflextemplatejob/batchdataflowflextemplatejob-direct/create.yaml#L20)
+
+This will override the `cnrm.cloud.google.com/dcl2crd: "true"` or `cnrm.cloud.google.com/tf2crd: "true"` annotations in the CRD and enable the direct controller for the new test cases. The previously-existing test cases will continue to use the TF/DCL-based controller.
+
+Verify the new `-direct` test cases have equivalent behavior to the existing test cases (though not necessarily the exact same API interactions, if the direct controller achieves the same end result differently than the TF/DCL-based controller).
```

Comment:
Personally, I feel like it's impossible to expect external contributers to develop new controllers with only mock tests. They will need to verify behavior with real gcp, using `hack/record-gcp`.

Reply:
How about suggesting users to use `git mv` when creating the `-direct` test cases. So that the PR git-diff shows whether it gives equivalent behavior or not.